modules/security/pages/password-policy.adoc

```diff
@@ -1,32 +1,121 @@
+| Require at Least One special character .
+| true
+| true/false
+
+| `Security.UserPasswordPolicy.ExpirationDay`
+| How long the password remains valid after it has been updated
+| 90
+| [1, 2147483647]
+
+| `Security.UserPasswordPolicy.PasswordReuseThreshold`
+| What is the minimum number of unique passwords required before a previously used password can be reused
+| 5
+| [1, 20]
+|===
+
+
+Follows are the examples of passworld policy configuration:
+
+[.wrap,console]
+----
+$ gadmin config set Security.UserPasswordPolicy.PasswordReuseThreshold 2
+$ gadmin config set Security.UserPasswordPolicy.MinLength 8
+$ gadmin config set Security.UserPasswordPolicy.UppercaseLetterRequired true
+$ gadmin config set Security.UserPasswordPolicy.LowercaseLetterRequired true
+$ gadmin config set Security.UserPasswordPolicy.DigitRequired true
+$ gadmin config set Security.UserPasswordPolicy.SpecialCharacterRequired true
+$ gadmin config set Security.UserPasswordPolicy.ExpirationDay 2
+$ gadmin config set Security.UserPasswordPolicy.PasswordReuseThreshold 2
+$ gadmin config apply
+$ gadmin restart gsql
+----
+
+If we configure the values as above, here are some usage examples:
+
+*Case 1:* Wrong password
+
+- Attempt to alter the password using "abc!1234", which doesn't follow the policy `Security.UserPasswordPolicy.UppercaseLetterRequired` because this configured value is true.
+
+*Case 2:* The minimum number of unique passwords required before a previously used password can be reused not reached
+
+- The `Security.UserPasswordPolicy.PasswordReuseThreshold` is configured as 2. If the user first sets the password to "Xiaoming@1001" and then changes it to "Xiaoming@1002", these two operations will be successful as they follow the policy. However, if the password is changed back to "Xiaoming@1001", it will fail because it doesn't follow the policy. Since after setting the password to "Xiaoming@1001", there is only 1 change (from "Xiaoming@1001" to "Xiaoming@1002").
+
+*Case 3:* Password expiration is approaching
+
+- The value of `Security.UserPasswordPolicy.ExpirationDay` is 2, which is less than 7. A warning message will be displayed if the user logs in from the gsql client or graph studio.
+
+*Case 4:* Password has expired
+
+- If the password "Xiaoming@1001" was created 4 days ago, which is greater than `Security.UserPasswordPolicy.ExpirationDay` which is 2, so the user's password has expired.
+
+== what will happen if password has expired
+
+If the password has expired. The user need change the password use the command `gsql -u <username> -p <password> [--targetuser <target_username>] --changepassword <new_password>` or change the password in GraphStudio first, otherwise any operation which requires password authorization will be block.
+
+when the password will be expired?
+
+For example, if the password is created now, and we configure it with command `gadmin config set Security.UserPasswordPolicy.ExpirationDay 2` . The passsword will be expired in 2 days.
```

Comment:
Same as above , avoid usage of "we".
Change to : For example, if a password is created now and the command gadmin config set Security.UserPasswordPolicy.ExpirationDay 2 is used, the password will expire in 2 days.